flytekit/sdk/types.py

```diff
@@ -395,6 +419,52 @@ def assert_and_create(wf_params, a, b):
             )
     """
 
+    GenericProto = staticmethod(_proto.create_generic)
+    """
+    GenericProto type wraps a protobuf type to provide interoperability between protobuf and flyte typing system. Using
+    this type, you can define custom input/output variable types of flyte entities and continue to provide strong typing
+    syntax. Proto type serializes proto objects as binary (leveraging `flyteidl's Binary literal <https://github.com/lyft/flyteidl/blob/793b09d190148236f41ad8160b5cec9a3325c16f/protos/flyteidl/core/literals.proto#L63>`_).
+    A generic proto is a specialization of the Generic type with added convenience functions to support marshalling/
+    unmarshalling of the underlying protobuf object using the protobuf official json marshaller. While GenericProto type
+    does not produce the most space-efficient representation of protobufs, it's a suitable solution for making protobufs
+    easily accessible (i.e. humanly readable) in other flyte components (e.g. console, cli... etc.).
+
+    .. note::
+
+        The protobuf Python library should be installed on the PYTHONPATH to ensure the type engine can access the
+        appropriate Python code to deserialize the protobuf.
+
+    When used with an SDK-decorated method, expect this behavior from the default type engine:
+
+        As input:
+            1) If set, a Python protobuf object of the type specified in the definition.
+            2) If not set, a None value.
+
+        As output:
+            1) A Python protobuf object matching the type specified by the users.
+            2) Set None to null the output.
+
+        From command-line:
+            A base-64 encoded string of the serialized protobuf.
```

Comment:
nit: I am not sure if I understand what this is trying to convey

Reply:
If you specify this type as an input parameter or a Workflow and you want to launch the workflow from CMD (like through `flyte-cli execute-launch-plan`) this tells you how to pass the value

Reply:
I see